tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+- name: Set login shell for user www-data
+  user: name=www-data shell=/bin/bash
+
+- name: Clone the opencraft repository
+  git: repo=https://github.com/open-craft/opencraft.git dest="{{ opencraft_root_dir }}" update=no
+  become_user: www-data
+
+- name: Install system dependencies
+  command: make install_system_dependencies chdir="{{ opencraft_root_dir }}"
+
+- name: Install virtualenv
+  pip: name=virtualenv executable=pip3
+
+- name: Install python requirements
+  pip:
+    requirements="{{ opencraft_root_dir }}/requirements.txt"
+    virtualenv="{{ opencraft_virtualenv_dir }}"
+    virtualenv_python=python3.4
```

Comment:
This PR will need to be reconciled with the upgrade - cf https://github.com/open-craft/opencraft/pull/6